### ISSUE TITLE
Weird GPU usage from the launcher - Windows 10

### ISSUE THREAD

**Issue by Klaim** [renpy/renpy#1362]
Ren'Py 6.99.14.3135

On Windows 10 using the new task scheduler interface showing the gpu usage: when running the launcher (which seem to select my NVidia GTX 980M, not the Intel Integrated thing), the launcher process takes 10% GPU "3D" time. It starts low, around 2% but gradually reach 10% in less than 10 seconds.
In parallel of this I can see the "System" process doing 6 to 7% "Copy" operation on the same GPU, until both tabilize and "System" stay at 2.5% copy operations and the launcher stays at 10-11% "3D" operations (always on the same GPU).
I observe that while doing nothing other than launch the GPU.

I am surprised that the launcher does anything like this.
Is there an issue from the recent improvements?

**Comment by renpytom** [comment]
08a2b348bfa9eeddc3f32a713f9456a5243cd719 will put the launcher in powersave mode. 

Since 6.99.14, Ren'Py has been preferring framerate to power savings on systems with AC power, which will put more load on the GPU. You can change this by hitting shift+G and setting powersave to enable.

**Comment by Klaim** [comment]
Thanks, it seems to work as expected. Would it make sense to also make it use the software renderer as acceleration doesn't seem necessary to me for the launcher?

**Comment by renpytom** [comment]
No. The software renderer is going away
